exporter/stackdriverexporter/stackdriver.go

```diff
@@ -21,57 +21,90 @@ import (
 	"fmt"
 
 	"contrib.go.opencensus.io/exporter/stackdriver"
-	"go.opencensus.io/trace"
+	cloudtrace "github.com/GoogleCloudPlatform/opentelemetry-operations-go/exporter/trace"
 	"go.opentelemetry.io/collector/component"
 	"go.opentelemetry.io/collector/component/componenterror"
-	"go.opentelemetry.io/collector/consumer/consumerdata"
+	"go.opentelemetry.io/collector/consumer/pdata"
+	"go.opentelemetry.io/collector/consumer/pdatautil"
 	"go.opentelemetry.io/collector/exporter/exporterhelper"
+	traceexport "go.opentelemetry.io/otel/sdk/export/trace"
 	"google.golang.org/api/option"
 	"google.golang.org/grpc"
 )
 
-// stackdriverExporter is a wrapper struct of Stackdriver exporter
-type stackdriverExporter struct {
-	exporter *stackdriver.Exporter
+const name = "stackdriver"
+
+// traceExporter is a wrapper struct of OT cloud trace exporter
+type traceExporter struct {
+	texporter *cloudtrace.Exporter
+}
+
+// metricsExporter is a wrapper struct of OC stackdriver exporter
+type metricsExporter struct {
+	mexporter *stackdriver.Exporter
+}
+
+func (*traceExporter) Name() string {
+	return name
 }
 
-func (*stackdriverExporter) Name() string {
-	return "stackdriver"
+func (*metricsExporter) Name() string {
+	return name
 }
 
-func (se *stackdriverExporter) Shutdown(context.Context) error {
-	se.exporter.Flush()
-	se.exporter.StopMetricsExporter()
+func (te *traceExporter) Shutdown(context.Context) error {
+	// uncomment when flush is implemented in trace exporter
+	// te.texporter.Flush()
```

Comment:
https://github.com/GoogleCloudPlatform/opentelemetry-operations-go/pull/72 has been merged, you should be able to invoke flush now.